pkg/controllermanager/controller/seed/lifecycle/reconciler.go

```diff
@@ -133,15 +132,19 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		return reconcile.Result{RequeueAfter: r.Config.SyncPeriod.Duration}, nil
 	}
 
-	var lastTransitionTime time.Time
```

Comment:
Can it be written as 
```golang
	var gardenletOfflineSince interface{}
	if conditionGardenletReady != nil {
		gardenletOfflineSince = conditionGardenletReady.LastTransitionTime.UTC()
	} else {
		gardenletOfflineSince = "unknown"
	}
	log.Info("Gardenlet has not sent heartbeats for at least the configured shoot monitor period, setting shoot conditions and constraints to 'Unknown' for all shoots on this seed",
		"gardenletOfflineSince", gardenletOfflineSince,
		"now", r.Clock.Now().UTC(),
		"shootMonitorPeriod", r.Config.ShootMonitorPeriod.Duration,
	)
```

I think this way we duplicate much less things.

Reply:
Thanks, this is better.